pool/database/seed/seed_user.ml

```diff
@@ -15,6 +15,38 @@ type person =
 
 type persons = person list [@@deriving show, yojson]
 
+let known_users =
+  let create_contact i =
```

Comment:
why is this needed? We do have a random user generator.

Reply:
Yes, but as the users are randomly generated, tests can randomly fail (for example if the matcher test does not find any contacts for an experiment).

Reply:
That is possible, but it would make the tests depending on each other. So if we would change the order for example, it would break the test cases. What about adding an additional seed to the `setup_test` function?